### ISSUE TITLE
Provide non color-only visual indications for the TOC of the Clarity wizard

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#3975]
## Summary

The current step of the Clarity wizard is currently indicated only with different color of the current page indicator in the TOC

* What is the change?
Add non - color indication (underline or icon) for the current page of the wizard 
* Why should it go in Clarity?
Meets accessibility guidelines that state should not be indicated only with color
* Does this change impact existing behaviors? If so how?
No, only visuals
* If this change introduces a new behavior, is this behavior accessible?
Yes

## Use case

As a poor vision/ colorblind user I want to be able to understand which one is the current page of the wizard

## Examples
No

## Workarounds
No

**Comment by gnomeontherun** [comment]
I need more detail here, are you talking about how Step 2 is displayed like in this image?

![image](https://user-images.githubusercontent.com/79500/68146369-784cf080-fefd-11e9-8d0e-6ffcbc03293c.png)

The titles are bolded as well, so we need more information and a specific demo to talk through, and we can talk with our accessibility team then.

**Comment by bashibozuk** [comment]
Exactly, the color of the selected item is with insufficient contrast compared to the rest of the elements around it and to be honest if you did not point out that the title is bold I would not notice and I not a user with restricted eye sight

**Comment by brian-free** [comment]
We've tried using checkboxes for successfully completed steps.
Maybe an error icon for the invalid steps.
Blue for the current step.

![image](https://user-images.githubusercontent.com/43282191/70664927-a7573000-1c28-11ea-97d4-4655e8b8240c.png)

**Comment by bdryanovski** [comment]
Something to keep in mind is that not everyone use the wizard as step by step flow component. Some of the teams use it as modal with page selector to group settings or organize big views. So the icon solution have to be optional for them.

**Comment by brian-free** [comment]
Interesting, we created our own component for that use case "Modal with Nav".
Did you have to create a bunch of custom CSS to remove the page steps and left bar?

**Comment by gnomeontherun** [comment]
This is a duplicate of #3976 so closing this in favor of that one.
